src/__tests__/renderHook.test.js

```diff
@@ -0,0 +1,63 @@
+import { useState, useEffect } from 'react';
+import { renderHook } from '../renderHook';
+
+describe('renderHook', () => {
+  test('should call the hook', () => {
+    const spy = jest.fn();
+    renderHook(spy);
+    expect(spy).toHaveBeenCalledTimes(1);
+  });
+
+  test('should trigger cleanup behavior on unmount', () => {
+    const hookCleanup = jest.fn();
+    const { unmount } = renderHook(() => {
```

Comment:
can you extract these inline hooks to named custom ones? tests serve as documentation and it would be easier for users to adapt if they see how to test imported hook